src/components/NationalText/utils.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment } from 'react';
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last, sum, isFinite } from 'lodash';
+import moment from 'moment';
+import {
+  formatPercent,
+  formatDecimal,
+  getPercentChange,
+  formatEstimate,
+} from 'common/utils';
+
+const getTwoWeekPercentChange = (series: any[]): number | null => {
+  const lastTwoWeeks = series.slice(-14);
```

Comment:
I think you need `.slice(-15)` to get the two week delta since to get a 1-day delta you'd actually have to do `.slice(-2)` [which gives you the last 2 values (yesterday and today)].